ETM_Sketch/data/deform.glsl

```diff
@@ -0,0 +1,44 @@
+#ifdef GL_ES
+precision mediump float;
+precision mediump int;
+#endif
+
+#define M_PI 3.1415926535897932384626433832795
+
+#define PROCESSING_TEXTURE_SHADER
+
+uniform sampler2D texture;
+
+uniform float time;
+uniform vec2 resolution;
+uniform vec2 mouse;
+uniform vec4 offset;
+
+varying vec4 vertTexCoord;
+varying vec4 vertColor;
+
+void main(void) {
+  vec2 v = vertTexCoord.st;
+  vec2 st = vertTexCoord.st;
+  st.x *= 1;
+  st.y *= 3;
+	// st.x += step(0.5, mod(st.x,.000)) * 0.500;
+  st = fract(st);
```

Comment:
das sind letzlich alles so shader mathe sachen die ich aus den beispielen von book of shader genommen habe: https://thebookofshaders.com/glossary/?search=fract